nbs/01a_losses.ipynb

```diff
@@ -300,7 +304,21 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "AssertionError",
+     "evalue": "Torch not compiled with CUDA enabled",
```

Comment:
I think this error is being caused automatically through some github workflow action?
I created three different forks and pushed the same code without running this cell hoping it would go away but it didn't. Apologies if I'm missing something obvious here.

Reply:
Comment out the cell and run it to remove the outputs, and then uncomment. The cell won't be executed in CI

Reply:
Done in 8bd7e9f9d4ea53127dbae2beb122daf7549a624e